src/shared/components/Checkout/Checkout.styles.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react'
+import styled from '@emotion/styled'
+import { colors } from '@/shared/theme'
+import { Icon } from '..'
+import { css } from '@emotion/react'
+
+export const Container = styled.div`
+  padding: 24px;
+  width: 312px;
+  background-color: ${colors.gray[800]};
+  color: ${colors.white};
+`
+
+export const ProgressbarContainer = styled.div`
+  width: 24px;
+  height: 24px;
+`
+
+export const StepsProgressContainer = styled.div`
+  display: flex;
+  gap: 12px;
+  align-items: center;
+`
+
+export const TopRowContainer = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 32px;
+`
+
+export const StepsContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 14px;
```

Comment:
...and this one

Reply:
You right, wonder why the search didnt bring them out for me ty 👨‍💻